internal/engine/compiler/impl_amd64.go

```diff
@@ -1229,7 +1229,7 @@ func (c *amd64Compiler) compileCtz(o *wazeroir.OperationCtz) error {
 		return err
 	}
 
-	if runtime.GOOS != "darwin" {
+	if runtime.GOOS != "darwin" && runtime.GOOS != "freebsd" {
```

Comment:
maybe this is not need and i guess only change on ^ `compileClz ` was necessary but not sure